<launch>
	<arg name="debug" default="true"/>

	<group>
		<node
			pkg="bixi_vision"
			type="arrow_detection"
			name="arrow_detection"
			required="true"
			output="screen"
		>
		<param name="subscribed_image_topic" value="image_rect_color"/>
		<param name="debug" value="$(arg debug)"/>
		<param name="published_topic" value="arrow"/>

		</node>
	</group>
</launch>
